vllm/model_executor/models/utils.py

```diff
@@ -597,12 +597,15 @@ def make_empty_intermediate_tensors_factory(keys: List[str], hidden_size: int):
 
     def make_empty_intermediate_tensors(
         batch_size: int,
```

Comment:
Did we check if the change of the return tensor of "make_empty_intermediate_tensors" does not break other models like Mixtral?